sqlshell.cmd

```diff
@@ -0,0 +1,14 @@
+@echo off

```

Comment:
That would be a Splice developer sqlshell on Windows. Only there are no Splice developers on Windows.
I think this file should be removed for now.

Reply:
You're right, I will remove it (and revert the comment in GETTING-STARTED.md